docs/ruleset_functions/get_HVAC_systems_primarily_serving_comp_rooms.md

```diff
@@ -1,35 +1,84 @@
+                    
+                Interior lighting Wattage calculation: 
+                - Get list of interior lighting objects: `lgting_obj_list = list(space.interior_lighting)`   
+                - Reset the temp_total_power variable: `temp_total_power = 0`  
+                - For each interior lighting object: `for int_lgt in lgting_obj_list:`  
+                    - Get the multiplier schedule: `multiplier_sch = get_component_by_id(B_RMI,int_lgt.lighting_multiplier_schedule)`  
+                    - Get the design_cooling_multiplier_schedule: `design_cooling_multiplier_sch = multiplier_sch.cooling_design_day_sequence`  
+                    - Get the maximum value in the schedule (the schedule should have this value for every hour): `max_sch = max(design_cooling_multiplier_sch)`  
+                    - Get the power per area for the lighting object: `lgt_W_sf = int_lgt.power_per_area`  
+                    - Calculate the power for the lighting object including schedule fraction: `lgt_W = lgt_W_sf * area * max_sch `  
+                    - Add to total across the space: `temp_total_power = temp_total_power + lgt_W`  
+                - Add to the running Wattage total for the space: `total_Wattage_space = total_Wattage_space + temp_total_power`  
+                    
+                Miscellaneous Wattage calculation:  
+                - Get list of misc equipment objects: `misc_obj_list = list(space.miscellaneous_equipment)`        
+                - Reset the temp_total_power variable: `temp_total_power = 0`                  
+                - For each misc equipment object: `for misc in misc_obj_list:`  
+                    - Get the multiplier schedule: `multiplier_sch = get_component_by_id(B_RMI,misc.multiplier_schedule)`  
+                    - Get the design_cooling_multiplier_schedule: `design_cooling_multiplier_sch = multiplier_sch.cooling_design_day_sequence`  
+                    - Get the maximum value in the schedule (the schedule should have this value for every hour): `max_sch = max(design_cooling_multiplier_sch)`  
+                    - Get the power for the misc object: `misc_power = misc.power`  
+                    - Calculate the power for the misc object including schedule fraction and sensible fraction: `misc_W = misc_power * max_sch * misc.sensible_fraction `  
+                    - Add to total across the space: `temp_total_power = temp_total_power + misc_W`  
+                - Add to the running Wattage total for the space: `total_Wattage_space = total_Wattage_space + temp_total_power`  
+            
+            - Add to the total Wattage across all spaces serving zone: `total_Wattage_zone = total_Wattage_zone + total_Wattage_space`    
+            - Check if the space serves a computer room: `if space_is_a_computer_room == true: total_Wattage_including_computer_rooms_only_zone = total_Wattage_including_computer_rooms_only_zone + total_Wattage_space`                     
```

Comment:
total_Wattage_including_computer_rooms_only_zone make is a little confusing. Shouldn't it be total_Wattage_of_computer_rooms_only?

Reply:
Modified name